packages/core/bootstrap/src/lib/cache-warmer/epics.ts

```diff
@@ -174,18 +174,44 @@ export const warmupRequestHandler: Epic<AnyAction, AnyAction, any> = (action$, s
     filter(warmupRequested.match),
     // fetch our required state to make a request to warm up an adapter
     withLatestFrom(state$),
-    map(([action, state]) => ({
-      requestData: state.cacheWarmer.subscriptions[action.payload.key],
-      key: action.payload.key,
-    })),
+    map(([action, state]) => {
+      const requestData = state.cacheWarmer.subscriptions[action.payload.key]
```

Comment:
We could using a nullish coalescing operator to avoid doing `Object.keys` twice.

```suggestion
      const requestData = state.cacheWarmer.subscriptions[action.payload.key] ?? {}
      const childKeys = Object.keys(requestData.childLastSeenById ?? {})
```